backend/app/api/v1/endpoints/health_check.py

```
@@ -1,19 +1,15 @@
 from fastapi import APIRouter, Response
 from app.utils.response_handler import ResponseHandler
+from app.core.exceptions.http_exception import BadRequestException
 
 router = APIRouter()
 
 @router.get("")
 async def health_check(response: Response):
```

Ruff (0.14.10)

Unused function argument: `response`

(ARG001)